tools/downloader/requirements-pytorch.in

```diff
@@ -0,0 +1,4 @@
+onnx
+scipy
```

Comment:
Could you add a comment about why `scipy` is required?

Reply:
This is torchvision's requirement, at least one model (inception/googlenet v3) use it

Reply:
Yeah, _I_ know that. But people who read this later won't. That's why I'm asking you to add a comment.